README.md

```diff
@@ -36,71 +36,59 @@ It supports several backends ([Docker](https://www.docker.com/), [Swarm mode](ht
 
-Traditional reverse-proxies are not natively dynamic. You can't change their configuration and hot-reload easily.
+Træfik listens to your service registry/orchestrator API and instantly generates the routes so your microservices are connected to the outside world -- without further intervention from your part. 
 
-Here enters Træfik.
+**Run Træfik and let it do the work for you!** 
+_(But if you'd rather configure some of your routes manually, Træfik supports that too!)_
 
 ![Architecture](docs/img/architecture.png)
 
-Træfik can listen to your service registry/orchestrator API, and knows each time a microservice is added, removed, killed or upgraded, and can generate its configuration automatically.
-Routes to your services will be created instantly.
-
-Run it and forget it!
-
-
 ## Features
 
-- [It's fast](https://docs.traefik.io/benchmarks)
-- No dependency hell, single binary made with go
-- [Tiny](https://microbadger.com/images/traefik) [official](https://hub.docker.com/r/_/traefik/) docker image
-- Rest API
-- Hot-reloading of configuration. No need to restart the process
+- Continuously updates its configuration (No restarts!)
+- Load balancer, Round Robin, rebalancer load-balancers
+- Enables HTTPS on your microservice (thanks to its [Let's Encrypt](https://letsencrypt.org) support)
 - Circuit breakers, retry
-- Round Robin, rebalancer load-balancers
-- Metrics (Rest, Prometheus, Datadog, Statsd, InfluxDB)
-- Clean AngularJS Web UI
-- Websocket, HTTP/2, GRPC ready
-- Access Logs (JSON, CLF)
-- [Let's Encrypt](https://letsencrypt.org) support (Automatic HTTPS with renewal)
-- [Proxy Protocol](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt) support
 - High Availability with cluster mode (beta)
-
-## Supported backends
-
-- [Docker](https://www.docker.com/) / [Swarm mode](https://docs.docker.com/engine/swarm/)
-- [Kubernetes](https://kubernetes.io)
-- [Mesos](https://github.com/apache/mesos) / [Marathon](https://mesosphere.github.io/marathon/)
-- [Rancher](https://rancher.com) (API, Metadata)
-- [Consul](https://www.consul.io/) / [Etcd](https://coreos.com/etcd/) / [Zookeeper](https://zookeeper.apache.org) / [BoltDB](https://github.com/boltdb/bolt)
-- [Eureka](https://github.com/Netflix/eureka)
-- [Amazon ECS](https://aws.amazon.com/ecs)
-- [Amazon DynamoDB](https://aws.amazon.com/dynamodb)
-- File
-- Rest API
+- See the magic through its clean web UI
+- Websocket, HTTP/2, GRPC ready
+- Provides metrics (Rest, Prometheus, Datadog, Statsd, InfluxDB)
+- Keeps access logs (JSON, CLF)
+- [Fast](https://docs.traefik.io/benchmarks) ... which is nice
+- Exposes a Rest API
+- Packaged as a single binary file (made with :heart: with go) and available as a [tiny](https://microbadger.com/images/traefik) [official](https://hub.docker.com/r/_/traefik/) docker image
+
+
+## Supported Backends
+
+- [Docker](docs/configuration/backends/docker/) / [Swarm mode](docs/configuration/backends/docker/#docker-swarm-mode)
```

Comment:
Any particular reason some of these are 2 to a line, some just 1, and others more?
should we make this a proper list, or table?

Reply:
I've kept the initial list there. Not sure about what to do. I'll come up with a proposal. The table might be a good idea.

Reply:
@ldez thinks the way things are organised in the list is logical ... 

Docker / Swarm belong together, as do Mesos / Marathon.
Consul, Etcd, Zookeeper & BoltDB are KV stores.

I kind of agree with him now, especially since I'm not able to find something better ... WDYT?

Reply:
I'm ok, its a pretty nitpicky thing :P 

Lets get this ship sailing.